.gitignore

```diff
@@ -15,8 +15,8 @@
 .ensime_cache/
 .ensime_lucene
 .generated-mima*
-# The star is required for further !.idea/ to work, see https://git-scm.com/docs/gitignore
-.idea/*
+.idea/
+# SPARK-35223: Add IssueNavigationLink to make IDEA support hyperlink on JIRA Ticket and GitHub PR on Git plugin.
 !.idea/vcs.xml
 python/.idea
```

Comment:
@gengliangwang can you remove both:

```
!.idea/vcs.xml
python/.idea
```

and add a comment here https://github.com/apache/spark/blob/master/.idea/vcs.xml#L17 that the file should be fixed with -f option?

Reply:
+1 for comments of `-f`

Reply:
@HyukjinKwon I added the comment in `.gitignore` instead. The file vcs.xml is already in VCS so any changes will be caught by Git already.

Reply:
👌